src/pages/FindWorkflow/FindWorkflowModal.js

```diff
@@ -128,9 +154,46 @@ const FindWorkflowModal = ({ onDismiss }) => {
       ]),
       isSubHeaderActive('browse-suggested-workflows') && div({ style: { overflowY: 'auto', flexGrow: 1, display: 'flex', flexDirection: 'column', paddingLeft: '20px' } }, [
         div({ style: { display: 'flex', flexWrap: 'wrap', overflowY: 'auto', paddingBottom: 5, paddingLeft: 5 } }, [
-          _.map(method => h(MethodCard, { method, onClick: () => submitMethod({ method }), key: method.method_name }), suggestedWorkflowsList)
+          _.map(method => h(MethodCard, { method, onClick: () => submitMethod(method), key: method.method_name }), suggestedWorkflowsList)
         ])
-      ])
+      ]),
+      getConfig().isURLEnabled && isSubHeaderActive('add-a-workflow-link') && div({ style: { marginLeft: '4rem' } }, [
+        div({ style: { width: 500 } }, [h2(['Workflow Link'])]),
```

Comment:
TOL: This file is just on the edge of becoming too long to understand all in one go. Rather than adding this all inline, might it make sense to have a `ImportGithub` class that can (1) keep the code complexity in a separate file and (2) be tested as a unit, instead of as part of a larger `FindWorkflowModal` component?